src/dBFT/dBFT/config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "PluginConfiguration": {
+    "RecoveryLogs": "ConsensusState",
+    "IgnoreRecoveryLogs": false
```

Comment:
`"StartConsensus": true`?

Reply:
But how do you pass the wallet?

Reply:
An encapsulated wallet such as `RPCServer DummyWallet`?

Reply:
But perhaps we will need to pass the path and password duplicated here...

Reply:
What are you suggesting, @erikzhang ?

Reply:
We can register the wallet as a service. But maybe we should do it in another pr.

Reply:
will we control the start consensus in the plugin config instead of neo-cli config?

Reply:
I think so, @cloud8little, control it on neo-cli config will be a little bit confusing in my opinion.
It looks like we need to do this extra effort here to create the wallet as a service as Erik mentioned.

Reply:
@erikzhang,
another way to do could be to create a function here (called `GetStartConsensus()) that return `true or false`.

On the neo-node we check if this plugin exist and call this function.
If `true` calls 

``cs
IConsensusProvider consensus = Plugin.Plugins.OfType<IConsensusProvider>().FirstOrDefault();
consensus?.Start(wallet);
``